libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java

```diff
@@ -81,6 +81,20 @@ default List<String> showTables(String schemaName) {
      */
     List<DBObjectIdentity> listTables(String schemaName, String tableNameLike);
 
+    /**
+     * Show all external table names list in the specified schema
+     */
+    default List<String> showExternalTables(String schemaName) {
+        return showExternalTablesLike(schemaName, null);
+    }
+
+    List<String> showExternalTablesLike(String schemaName, String tableNameLike);
```

Comment:
`    Map<String, List<DBTableColumn>> listBasicColumnsInfo(String schemaName);`
should we modify the implementation of this method for external table?

Reply:
I think it's better to use map, but change the existing code

Reply:
what I mean is that `listBasicColumnsInfo` is designed for table and view, should we do any refactor to make it adapt for external table?

Reply:
ok